test/tests/pbs_smoketest.py

```diff
@@ -1466,11 +1466,12 @@ def test_man_pages(self):
         pbs_conf = self.du.parse_pbs_config(self.server.shortname)
         manpath = os.path.join(pbs_conf['PBS_EXEC'], "share", "man")
         pbs_cmnds = ["pbsnodes", "qsub"]
+        os.environ['MANPATH'] = '%s' % manpath 
```

Comment:
You don't need to do string formatting for just '%s'.  You can just directly set os.environ['MANPATH'] = manpath.